.circleci/config.yml

```diff
@@ -419,3 +464,13 @@ workflows:
     when: << pipeline.parameters.translation_review_build >>
     jobs:
       - translation-review-build
+  Release Build:
+    unless: << pipeline.parameters.translation_review_build >>
+    jobs:
+      - gutenberg-bundle-build
+      - Release Build:
+          filters:
+            branches:
+              ignore: /.*/
+            tags:
+              only: /^\d+(\.\d+)*(-rc-\d)?$/
```

Comment:
Yup! This popped out in sister PRs in other repos as well. 
My reasoning for keeping it this way is that this is a release action, so I'd prefer not to have it triggered by common testing patterns (like -rc-XX(X) usually are). 

I know that this can be initially confusing if we ever reach 10 release candidates, but this is so unlikely (it would mean 1 RC per day, including the last day of the sprint!) that I think it's worth it to keep this regex on the safe side. 

Does it make sense to you?

Reply:
Gotcha! Yeah it does make sense to me, totally.

The only thing I'd say, then, is to add the rationale for the pattern as a comment so the next reader of the file will know the reason why.

I love tools that use YAML instead of JSON for configurations for this exact reason: you can leave explanatory comments.